platforms/minigame/platforms/alipay/wrapper/builtin/version.txt

```diff
@@ -0,0 +1 @@
+bb1a532aaf865e28d4d5d2481e5e2a185e9c02e3
```

Comment:
What's the meaning of the hash value? Do we need the version.txt file?

Reply:
It's not need to integrate this file